cmd/api-linter/cli.go

```diff
@@ -105,14 +111,33 @@ func runCLI(rules lint.Rules, configs lint.Configs, args []string) error {
 				case "json":
 					marshal = json.Marshal
 				}
-				b, err := marshal(problems)
+				b, err := marshal(lintResponses)
 				if err != nil {
 					return err
 				}
-
 				if _, err = w.Write(b); err != nil {
 					return err
 				}
+
+				if c.BoolT("summary") {
```

Comment:
I would probably choose "summary" as another type of output. So json, yaml, or summary as options to --fmt.